tools/cordova/package-id-version-parser.js

```diff
@@ -0,0 +1,27 @@
+// Accepts a combined Cordova package ID + version string, then parses out
+// and returns the ID and version in a package details object.
+//
+// Example `packageIdAndVersion` formats:
+// some-cordova-plugin@1.0.0
+// @somescope/some-cordova-plugin@1.0.0
+exports.parse = packageIdAndVersion => {
+  const package = {};
+  if (packageIdAndVersion) {
+    const parts = packageIdAndVersion.match(
+      /^(@[^\/]*)?(\/)?([^@]*)(@)?(.*)?/
+    );
+    const details = {
+      scope: parts[1],
+      scopeSeparator: parts[2],
+      packageName: parts[3],
+      versionSeparator: parts[4],
```

Comment:
Maybe just not capture `versionSeparator`?

```js
/^(?:@([^\/]+)\/)?([^\/@]*)@?(.*)?/
```

Reply:
Sounds good!